mesheryctl/internal/cli/root/system/restart.go

```diff
@@ -97,3 +105,7 @@ func restart() error {
 	}
 	return nil
 }
+
+func init() {
+	restartCmd.Flags().BoolVarP(&skipUpdateFlag, "skip-update", "", false, "(optional) skip checking for new Meshery's container images.")
```

Comment:
We would only need to register the flag here I guess and it is centrally controlled in `start.go` for all the system commands.

Reply:
Got it. But won't we need to print during help flag used?

Reply:
I mean when --help is used....

Reply:
That's all. Only this was needed.

Reply:
Interesting....